### PR TITLE
Add amount_type ORGB for income breakdowns

### PR DESCRIPTION
**Pull request by michaelglenister**
Issue: https://trello.com/c/wswB6dfC/536-income-breakdown-is-not-shown-despite-mscoa-data-being-loaded-now-tb-1-day

### CODE REVIEW

File: scorecard/profile_data/api_data.py [hunk 1]
```diff
@@ -806,7 +806,7 @@ def get_queries(self):
                 "cube": "grants_v2",
                 "aggregate": "amount.sum",
                 "cut": {
-                    "amount_type.code": ["ACT", "SCHD", "TRFR"],
+                    "amount_type.code": ["ACT", "SCHD", "TRFR", "ORGB"],
```
[the remark this answers is not in the document]
Comment (michaelglenister) [reply scorecard/profile_data/api_data.py:809]: Technically correct but there is already code filtering ["SCHD", "TRFR", "ACT"] amount types for the national conditional grants indicator. Cleaned that up and there are no changes needed for income.js now

File: scorecard/tests/test_scorecard.py [hunk 2]
```diff
@@ -134,3 +135,33 @@ def test_data_explorer_links(self):
         self.assertIn('http://portal:8002/table/cflow/?municipalities=BUF&year=2019&items=3010%2C3030%2C3040%2C3050%2C3060%2C3070%2C3100&amountType=AUDA', element)
         element = selenium.find_elements_by_css_selector(link_class)[19].get_attribute("href")
         self.assertIn('http://portal:8002/table/incexp/?municipalities=BUF&year=2019&items=0200%2C0400%2C1000&amountType=AUDA', element)
+
+    def test_grants_section(self):
+        selenium = self.selenium
+        selenium.get("%s%s" % (self.live_server_url, "/profiles/municipality-BUF-buffalo-city/"))
+
+        element = selenium.find_element_by_css_selector('#income-summary').text
+        self.assertIn('2018-2019 Audited actual', element)
```
Comment (jbothma) [inline scorecard/tests/test_scorecard.py:144]: this is audit year

File: scorecard/tests/test_scorecard.py [hunk 3]
```diff
@@ -134,3 +135,33 @@ def test_data_explorer_links(self):
...
+        self.assertIn('R6 040 712 322', element)
+
+        element = selenium.find_element_by_css_selector('#types-of-transfers').text
+        self.assertIn('2018-2019 Original budget', element)
```
Comment (jbothma) [inline scorecard/tests/test_scorecard.py:148]: this is supposed to be audit year + 1

File: scorecard/tests/test_scorecard.py [hunk 4]
```diff
@@ -134,3 +135,33 @@ def test_data_explorer_links(self):
...
+        self.assertIn('R183 861 100', element)
+        element = selenium.find_element_by_css_selector('#w-dropdown-list-11').get_attribute("innerHTML")
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('2019-2020 Original budget', element)
+
+        element = selenium.find_element_by_css_selector('#equitable-share').text
+        self.assertIn('2018-2019 Original budget', element)
```
Comment (jbothma) [inline scorecard/tests/test_scorecard.py:155]: ditto

File: scorecard/tests/test_scorecard.py [hunk 5]
```diff
@@ -134,3 +135,33 @@ def test_data_explorer_links(self):
...
+        self.assertIn('R79 021 100', element)
+        element = selenium.find_element_by_css_selector('#w-dropdown-list-12').get_attribute("innerHTML")
```
Comment (jbothma) [inline scorecard/tests/test_scorecard.py:157]: this is not a robust selector to use - is there a class you can use to select "the webflow dropdown under #equitable-share"? like `#equitable-share .dropdown`

File: scorecard/tests/test_scorecard.py [hunk 6]
```diff
@@ -134,3 +135,33 @@ def test_data_explorer_links(self):
...
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('2019-2020 Original budget', element)
```
Comment (jbothma) [inline scorecard/tests/test_scorecard.py:159]: we need to test that when you select this, the data shown updates to match this year

File: scorecard/tests/test_scorecard.py [hunk 7]
```diff
@@ -134,3 +135,33 @@ def test_data_explorer_links(self):
...
+
+        element = selenium.find_element_by_css_selector('#national-conditional-grants').text
+        self.assertIn('2019-2020 Allocations', element)
+        self.assertIn('R285 942 000', element)
+
+        element = selenium.find_element_by_css_selector('#provincial-transfers').text
+        self.assertIn('2019-2020 Original budget', element)
```
Comment (jbothma) [inline scorecard/tests/test_scorecard.py:166]: how come this is the correct year, but in staging it's not?

File: scorecard/tests/test_scorecard.py (hunk 3 above)
Comment (michaelglenister) [reply scorecard/tests/test_scorecard.py]: Changed

File: scorecard/tests/test_scorecard.py (hunk 4 above)
Comment (michaelglenister) [reply scorecard/tests/test_scorecard.py]: Changed

File: scorecard/tests/test_scorecard.py (hunk 5 above)
Comment (michaelglenister) [reply scorecard/tests/test_scorecard.py]: Yes changed to `#equitable-share .dropdown`

File: scorecard/tests/test_scorecard.py (hunk 6 above)
Comment (michaelglenister) [reply scorecard/tests/test_scorecard.py:159]: Added a test for the dropdown selection

File: scorecard/tests/test_scorecard.py (hunk 7 above)
Comment (michaelglenister) [reply scorecard/tests/test_scorecard.py:166]: As we found out this was a result of the data not being provided - hopefully for a short time only

File: scorecard/tests/test_scorecard.py (hunk 5 above)
Comment (jbothma) [reply scorecard/tests/test_scorecard.py]: lol did I guess right?!

Comment (michaelglenister) [reply scorecard/tests/test_scorecard.py]: Yeah I wondered if it was a guess or a polite suggestion

**Pull request merged**